### DISCUSSION TITLE
[Feature] expose option for admins to unlink all users’ oauth accounts

### DISCUSSION THREAD

**Discussion by aclerici38** [immich-app/immich#14977] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently I believe the only way for an admin to unlink users’ oauth accounts is by manually updating the database. I believe there should be an “unlink all accounts’ oauth” or similar button in the admin ui to make easy transitions between oauth providers possible.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is now implemented
